pkg/aggregator/aggregator.go

```diff
@@ -804,20 +799,18 @@ func (agg *BufferedAggregator) run() {
 			// we can use the aggregator to buffer them
 			agg.addOrchestratorManifest(&orchestratorManifest)
 		case event := <-agg.eventPlatformIn:
-			state := stateOk
-			tlmProcessed.Add(1, event.eventType)
-			aggregatorEventPlatformEvents.Add(event.eventType, 1)
-			err := agg.handleEventPlatformEvent(event)
-			if err != nil {
-				state = stateError
-				aggregatorEventPlatformEventsErrors.Add(event.eventType, 1)
-				log.Debugf("error submitting event platform event: %s", err)
-				if !aggregatorEventPlatformErrorLogged {
+			go func(event senderEventPlatformEvent) {
+				state := stateOk
+				tlmProcessed.Add(1, event.eventType)
+				aggregatorEventPlatformEvents.Add(event.eventType, 1)
+				err := agg.handleEventPlatformEvent(event)
+				if err != nil {
+					state = stateError
+					aggregatorEventPlatformEventsErrors.Add(event.eventType, 1)
 					log.Warnf("Failed to process some event platform events. error='%s' eventCounts=%s errorCounts=%s", err, aggregatorEventPlatformEvents.String(), aggregatorEventPlatformEventsErrors.String())
-					aggregatorEventPlatformErrorLogged = true
```

Comment:
Note for reviewers: Removing `aggregatorEventPlatformErrorLogged` since it seems mainly used to avoid spam of logs like `event platform forwarder pipeline channel is full`. cc @djova

Reply:
Yeah, isn't that pretty important? Without it we could see tons of these logs?